server/README.md

```diff
@@ -11,6 +11,25 @@
 
 3. Run the application with: `sbt run`
 
+## Troubleshooting
+
+1. If when you run xsnd you get 
+
+    `error: Could not locate RPC credentials.`
+
+    create a file `~/.xsncore/xsn.conf` with
+    ```
+    rpcuser=dummy
+    rpcpassword=replaceme
+    rpcport=51473
+    txindex=1
+    ```
+    after this step, you can run it with just `bin/xsnd`
+
+2. If you have connection problems with xsnd node, try disabling IPv6.
```

Comment:
If the `xsnd` node is not getting any blocks/peers, try disabling IPv6, and remove the peers file (`~/.xsncore/peers.dat`), restarting `xsnd` should get peers now.